hack/prowimagebuilder/main.go

```diff
@@ -223,26 +223,31 @@ func buildAndPush(id *imageDef, koDockerRepo string, push bool) error {
 	if err := setup(id); err != nil {
 		return fmt.Errorf("setup: %w", err)
 	}
-	if _, err = runCmd("_bin/ko", publishArgs...); err != nil {
-		return fmt.Errorf("running ko: %w", err)
+	// ko only supports a single docker repo at a time, running this repeatedly
+	// on different docker repos so that multiple docker repos can be supported.
+	// This process utilized the built in cache of ko, so that pushing to
+	// subsequent docker repo(s) is relatively cheap.
+	for _, dockerRepo := range dockerRepos {
+		if err = os.Setenv("KO_DOCKER_REPO", dockerRepo); err != nil {
```

Comment:
This is not threadsafe, right? (`buildAndPush` is called concurrently.)

Reply:
good catch, indeed
/hold

Reply:
updated to limit the scope of env var on command execution level, should be good now
/unhold
@cjwagner please take another look

Reply:
This still needs to be removed.

Reply:
good catch, removed, thank you!